cellxgene_resources/curation_visium.ipynb

```diff
@@ -0,0 +1,227 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#REQUIRED to distinguish single section data from integrated data\n",
+    "adata.uns['spatial']['is_single'] = True"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#PREFERRED to include fullres image\n",
+    "fullres_path = 'WS_PLA_S9101764.tif'\n",
+    "adata.uns['spatial'][library_id]['images']['fullres'] = np.asarray(Image.open(fullres_path))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "adata.uns['title'] = library_id"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#OPTIONAL to move metadata before deleting it\n",
+    "adata.uns['spatial_metadata'] = adata.uns['spatial'][library_id]['metadata']\n",
+    "\n",
+    "#MUST NOT SUBMIT metadata or lowres or fiducial_diameter_fullres in uns.spatial\n",
+    "del adata.uns['spatial'][library_id]['metadata']\n",
+    "del adata.uns['spatial'][library_id]['images']['lowres']\n",
+    "del adata.uns['spatial'][library_id]['scalefactors']['tissue_lowres_scalef']\n",
+    "del adata.uns['spatial'][library_id]['scalefactors']['fiducial_diameter_fullres']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#REQUIRED obs metadata - will be the same for all Visium Datasets\n",
+    "adata.obs['suspension_type'] = 'na'\n",
+    "adata.obs['assay_ontology_term_id'] = 'EFO:0010961'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#REQUIRED obs metadata - most likely the same value for all obs\n",
+    "#update based on the given donor/sample\n",
+    "adata.obs['donor_id'] = 'donor_A'\n",
+    "adata.obs['organism_ontology_term_id'] = 'NCBITaxon:9606' #NCBITaxon:9606 for human, NCBITaxon:10090 for mouse\n",
+    "adata.obs['sex_ontology_term_id'] = 'PATO:0000383' #PATO:0000383 for female, PATO:0000384 for male\n",
+    "adata.obs['development_stage_ontology_term_id'] = 'HsapDv:0000137' #HsapDv or MmusDv term\n",
+    "adata.obs['self_reported_ethnicity_ontology_term_id'] = 'HANCESTRO:0022' #HANCESTRO term\n",
+    "adata.obs['disease_ontology_term_id'] = 'PATO:0000461' #PATO:0000461 for normal, MONDO term for disease\n",
+    "adata.obs['tissue_type'] = 'tissue' #tissue, organoid\n",
+    "adata.obs['tissue_ontology_term_id'] = 'UBERON:0001115' #UBERON term"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#map in cell_types\n",
+    "#this example reads in a csv with barcodes in the first column and CL terms in the second column\n",
```

Comment:
And a little more to clarify that a csv in this format can be provided to add in the cell ontology but it is not part of the output files or found in this repo

Reply:
Added additional comments on this. Let me know what you think

Reply:
This looks good too, hopefully will be clear for contributors.